internal/rollup/rollup_bundle.bzl

```diff
@@ -117,14 +138,14 @@ def run_uglify(ctx, input, output, debug = False):
 
 def _rollup_bundle(ctx):
   rollup_config = write_rollup_config(ctx)
-  run_rollup(ctx, rollup_config, ctx.outputs.build_es6)
-  run_tsc(ctx, ctx.outputs.build_es6, ctx.outputs.build_es5)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min_debug, debug = True)
-  return DefaultInfo(files=depset([ctx.outputs.build_es5_min]))
+  output_dir_es6 = run_rollup(ctx, rollup_config)
+  output_dir_es5 = run_tsc(ctx, output_dir_es6)
+  #run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min)
+  #run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min_debug, debug = True)
+  return DefaultInfo(runfiles=ctx.runfiles([output_dir_es6, output_dir_es5]), files=depset([]))
 
 ROLLUP_ATTRS = {
-    "entry_point": attr.string(mandatory = True),
+    "entry_points": attr.string_list(mandatory = True),
```

Comment:
in the future we have to think about backward compat for these changes, and avoid breaking.
Maybe not yet? maybe soon? hard to say how many users we have